cognite/client/_api/extractionpipelineruns.py

```diff
@@ -0,0 +1,84 @@
+from typing import *
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import ExtractionPipelineRunFilter, ExtractionPipelineRunList
+from cognite.client.data_classes.extractionpipelineruns import ExtractionPipelineRun, StringFilter
+from cognite.client.data_classes.shared import TimestampRange
+
+
+class ExtractionPipelineRunsAPI(APIClient):
+    _RESOURCE_PATH = "/extpipes/runs"
+    _LIST_CLASS = ExtractionPipelineRunList
+
+    def list(
+        self,
+        external_id: str,
+        statuses: List[str] = None,
+        message_substring: str = None,
+        created_time: Union[Dict[str, Any], TimestampRange] = None,
+        limit: int = 25,
+    ) -> ExtractionPipelineRunList:
+        """List runs for an extraction pipeline with given external_id <>`_
```

Comment:
```suggestion
        """`List runs for an extraction pipeline with given external_id <>`_
```
nit but this conflicts with documentation generation